### PR TITLE
int: Web app rock paper scissors

### PR DESCRIPTION
**Pull request by nshanbhag20001412**
## Issue 16

### CODE REVIEW

File: Rock-Paper-Scissors/index.html [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    </div><br>
+
+    <div class="result">
+      <p></p>
+    </div>
+
+
+    <script src="index.js" charset="utf-8"></script>
+  </body>
+</html>
```
Comment (soumik2012) [inline Rock-Paper-Scissors/index.html:40]: Add a new line at EOF

File: Rock-Paper-Scissors/index.js [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+}
+
+function main() {
+  rock_div.addEventListener('click', () => game("r"));
+  paper_div.addEventListener('click', () => game("p"));
+  scissors_div.addEventListener('click', () => game("s"));
+}
+
+
+main();
```
Comment (soumik2012) [inline Rock-Paper-Scissors/index.js:85]: Add new line at the EOF

File: Rock-Paper-Scissors/style.css [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+  border: 4px solid #FC121B;
+  box-shadow: 0 0 10px #D01115;
+}
+
+.gray-glow {
+  border: 4px solid #464647;
+  box-shadow: 0 0 10px #25292B;
+}
+
+
```
Comment (soumik2012) [inline Rock-Paper-Scissors/style.css:131]: Remove all the extra line you made

**Review by soumik2012 — CHANGES_REQUESTED** [review]
Minor change needed then we are good to go!!

File: Rock-Paper-Scissors/index.js (hunk 2 above)
Comment (nshanbhag20001412) [reply Rock-Paper-Scissors/index.js:85]: didnt understand this

File: Rock-Paper-Scissors/style.css (hunk 3 above)
Comment (nshanbhag20001412) [reply Rock-Paper-Scissors/style.css:131]: you mean the spaces between each style component?

**Comment by nshanbhag20001412** [comment]
resolved

File: Rock-Paper-Scissors/style.css (hunk 3 above)
Comment (soumik2012) [reply Rock-Paper-Scissors/style.css]: Yup

**Review by soumik2012 — APPROVED** [review]
LGTM 💯

**Comment by soumik2012** [comment]
@nshanbhag20001412 Thank you for contributing to @Tenet-coding 🥳

**Pull request merged**